sympy/stats/symbolic_probability.py

```diff
@@ -109,17 +125,19 @@ def __new__(cls, expr, condition=None, **kwargs):
             condition = _sympify(condition)
             obj = Expr.__new__(cls, expr, condition)
         obj._condition = condition
+        obj._evaluate = kwargs.get('evaluate', True)
```

Comment:
It should not be necessary to store `evaluate`. If that is needed then I think there has been a misunderstanding.

Reply:
Okay!, I will remove evaluate, and add some internal flag which helps to get the result in integral form when called ` .rewrite(Integral)`